arkane/input.py

```diff
@@ -706,7 +706,7 @@ def process_model_chemistry(model_chemistry):
         # assume this is an sp//freq format, split
         sp_level, freq_level = model_chemistry.split('//')
     else:
-        # assume only the sp level was given in the model chemistry, assign to sp and return None for freq
+        # assume the sp freq levels are the same, assign to the model chemistry to both
```

Comment:
I think some words might have gotten lost here. You probably meant something like `assume the sp and freq levels are the same, assign the model chemistry to both`